src/test/test_private_sts.c

```diff
@@ -4,7 +4,7 @@
 
 #ifdef STS_COMPILE_UNIT_TESTS
 
-#include "test.h"
```

Comment:
Hm... Yes, it seems to be somewhat better approach, I must have misunderstood you the previous time. Fixing.